corehq/apps/accounting/invoicing.py

```diff
@@ -365,7 +365,8 @@ def __init__(self, date_start, date_end, account, recipients=None):
 
     def create_invoice(self):
         for sub in self.account.subscription_set.filter(do_not_invoice=False):
-            if should_create_invoice(sub, sub.subscriber.domain, self.date_start, self.date_end):
+            if (not sub.plan_version.plan.edition == SoftwarePlanEdition.COMMUNITY
+                    and should_create_invoice(sub, sub.subscriber.domain, self.date_start, self.date_end)):
```

Comment:
This revert seems unfortunate.  Does this mean that `DomainInvoiceFactory` does want to create invoices for community plans? In other words, is the "should create invoice" criteria actually fundamentally different? If not, could a simple refactor allow putting the community plan condition back into `should_create_invoice`?

Reply:
yes, DomainInvoiceFactory should create invoices for community plans if they have excess mobile workers or SMSes (possible with a few grandfathered plans)